Bottom: f94d043d236a493fc86bdf7cd4b25cfa3d060d8b
Top:    0354d490730c8c748e83053517e9d079ac584f2b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-30 14:59:09 -0500

Refresh of add-haproxy-role

---
diff --git a/roles/haproxy/defaults/main.yml b/roles/haproxy/defaults/main.yml
new file mode 100644
index 00000000..1fcca921
--- /dev/null
+++ b/roles/haproxy/defaults/main.yml
@@ -0,0 +1,10 @@
+---
+
+image_name: "haproxy:1.8.12-alpine"
+
+conf_volume_hostdir: "{{ docker_restore_config_base_dir }}/{{ container_name }}"
+conf_volume_mount: "/usr/local/etc/haproxy/"
+
+container_name: "{{ docker_name_prefix }}haproxy"
+container_volumes:
+  - "{{conf_volume_hostdir}}:{{conf_volume_mount}}:ro"
diff --git a/roles/haproxy/files/haproxy.cfg b/roles/haproxy/files/haproxy.cfg
new file mode 100644
index 00000000..5206b838
--- /dev/null
+++ b/roles/haproxy/files/haproxy.cfg
@@ -0,0 +1,23 @@
+global
+    daemon
+    maxconn 4096
+
+defaults
+    mode http
+    timeout connect 5000ms
+    timeout client 50000ms
+    timeout server 50000ms
+
+frontend http-in
+    bind 0.0.0.0:80
+    acl is_nexus       hdr(host)       eq nexus.novatech-llc.com
+    acl is_buildbot    hdr(buildbot)   eq buildbot.novatech-llc.com
+
+    use_backend nexus if is_nexus
+    use_backend buildbot if is_buildbot
+
+backend buildbot
+    server buildbot1 buildbot.local:8080 maxconn 32
+
+backend nexus
+    server nexus1 nexus.local:8081 maxconn 32
diff --git a/roles/haproxy/tasks/main.yml b/roles/haproxy/tasks/main.yml
new file mode 100644
index 00000000..e69de29b
